static.config.js

```diff
@@ -297,6 +297,11 @@ async function getRoutes() {
       getData: () => spec,
       template: "src/containers/single/api",
     },
+    {
+      path: "/product-news-old",
+      getData: async () => ({ productNews }),
+      template: "src/containers/single/product-news-old",
+    },

```

Comment:
```suggestion
    //{
    //  path: "/product-news-old",
    //  getData: async () => ({ productNews }),
    //  template: "src/containers/single/product-news-old",
    //},
```

No need to keep the old page visible